.github/workflows/test.yml

```diff
@@ -0,0 +1,12 @@
+name: my workflow
```

Comment:
I think you created this file for testing, could you please delete the file before merging?